create new action yamls with modified naming convention

kellyaa opened the pull request:
Addresses #60

kvmware commented:
So, the convention i used for `reusable_style.yaml` was an underscore but maybe the dash is better whichever you choose please just make it consistent. ie. rename `reusable_style.yaml` to `reusable-style.yaml`. There seems to be a bunch of linting errors https://github.com/knative/actions/runs/8003446781?check_suite_focus=true that we should try to address as well if possible, it might be too much and that is fine but at least we can make an effort to fix some of the low hanging ones.

kellyaa commented:
Thanks! I changed the naming to `reusable-style.yaml` and addressed most of the shellcheck linting errors. The ones I didn't address were in regards to having a description to the action yaml input variables. (need to get more info on that first)

kvmware commented:
/lgtm
/approve
